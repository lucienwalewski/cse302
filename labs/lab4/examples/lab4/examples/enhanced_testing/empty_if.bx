var x = 10, y = 15 : int; // Global variable to test type checking

def function(arg : int) {

print(x);
var x = 20 : int;

print(x);

if (true) {
    var x = 30 : int;
    print(x);

    if (true){
        var x = 40 : int;
        print(x);
        x = 50;
        print(x);
    }


        if (y != 15){
        
        }
        else {
            print(y);
        }





}

print(x);
    
    


}

var z = true : bool;

def main() {
  function(5);
  print(x);
  
  
}