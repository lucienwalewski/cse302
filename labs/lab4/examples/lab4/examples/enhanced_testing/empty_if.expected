10
20
30
40
50
15
20
10